node_type: eagerx_core.basenode/ActionsNode
default:
  inputs: [observations_set, step]
  outputs: [set]
  process: 1  # {0: NEW_PROCESS, 1: ENVIRONMENT, 2: BRIDGE, 3: EXTERNAL}
  color: red
inputs:
  observations_set:
    address: env/observations/outputs/set
    msg_type: std_msgs.msg/UInt64
    repeat: all
  step:
    address: env/supervisor/outputs/step
    msg_type: std_msgs.msg/UInt64
    repeat: all
outputs:
  set:
    msg_type: std_msgs.msg/UInt64
    start_with_msg: True
